Cely Demo/AppDelegate.swift

```diff
@@ -16,7 +16,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
-        Cely.setup(with: window, forModel: User.ref, requiredProperties:[.Token])
+
+        Cely.setup(with: window!, forModel: User.ref, requiredProperties: [.Token], withOptions: [
+            .LoginCompletionBlock: { (username: String, password: String) in
+                if username == "asdf" && password == "asdf" {
```

Comment:
Why do you do this check here? Login is completed already, so the check has been already done before this block is called, no?